src/controllers/invalid.company.status.controller.ts

```diff
@@ -3,10 +3,16 @@ import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/compa
 import { getCompanyProfile } from "../services/company.profile.service";
 import { Templates } from "../types/template.paths";
 import { URL_QUERY_PARAM } from "../types/page.urls";
+import { isCompanyNumberValid } from "../validators/company.number.validator";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const companyNumber = req.query[URL_QUERY_PARAM.COMPANY_NUM];
+    const companyNumber = req.query[URL_QUERY_PARAM.COMPANY_NUM] as string;
+
+    if (!isCompanyNumberValid(companyNumber)) {
+      return next(new Error("Invalid company number entered in invalid.company.status url query parameter"));
```

Comment:
Rather than reference 'invalid.company.status' here could you maybe use the INVALID_COMPANY_STATUS_PATH variable/constant? Just a thought.